### PR TITLE
feat: add building policies

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

File: src/Be.Vlaanderen.Basisregisters.AcmIdm/AddAuthorizationExtensions_MinimalApi.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
+namespace Be.Vlaanderen.Basisregisters.AcmIdm
```
Comment (lvermeulen) [inline src/Be.Vlaanderen.Basisregisters.AcmIdm/AddAuthorizationExtensions_MinimalApi.cs:1]: No underscores in file names

File: src/Be.Vlaanderen.Basisregisters.AcmIdm/AddAuthorizationExtensions_WebApi.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
+namespace Be.Vlaanderen.Basisregisters.AcmIdm
```
Comment (lvermeulen) [inline src/Be.Vlaanderen.Basisregisters.AcmIdm/AddAuthorizationExtensions_WebApi.cs:1]: No underscores in file names

File: src/Be.Vlaanderen.Basisregisters.AcmIdm/AuthorizationHandlers/AcmIdmAuthorizationHandler.cs [hunk 3]
```diff
@@ -10,8 +10,8 @@
 
     public class AcmIdmAuthorizationHandler : AuthorizationHandler<AcmIdmAuthorizationRequirement>
     {
-        private static readonly ReadOnlyDictionary<string, string> _cache =
-            new ReadOnlyDictionary<string, string>(new Dictionary<string, string>(1000));
+        // Todo: is this type thread safe when making concurrent writes to the dictionary cache?
```
Comment (lvermeulen) [inline src/Be.Vlaanderen.Basisregisters.AcmIdm/AuthorizationHandlers/AcmIdmAuthorizationHandler.cs:13]: doesn't matter, will be removed when we have a NisCode service

File: test/Be.Vlaanderen.Basisregisters.AcmIdm.Tests/IntegrationTests/IntegrationTests_MinimalApi.cs [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ﻿namespace Be.Vlaanderen.Basisregisters.AcmIdm.Tests.IntegrationTests
```
Comment (lvermeulen) [inline test/Be.Vlaanderen.Basisregisters.AcmIdm.Tests/IntegrationTests/IntegrationTests_MinimalApi.cs:1]: No underscores in file names

**Review by lvermeulen — COMMENTED** [review]
changes requested

File: test/Be.Vlaanderen.Basisregisters.AcmIdm.Tests/IntegrationTests/IntegrationTests_MinimalApi.cs (hunk 4 above)
Comment (jvandaal) [reply test/Be.Vlaanderen.Basisregisters.AcmIdm.Tests/IntegrationTests/IntegrationTests_MinimalApi.cs:1]: Changed to a dash.

File: src/Be.Vlaanderen.Basisregisters.AcmIdm/AuthorizationHandlers/AcmIdmAuthorizationHandler.cs (hunk 3 above)
Comment (jvandaal) [reply src/Be.Vlaanderen.Basisregisters.AcmIdm/AuthorizationHandlers/AcmIdmAuthorizationHandler.cs]: Removed comment.

File: src/Be.Vlaanderen.Basisregisters.AcmIdm/AddAuthorizationExtensions_WebApi.cs (hunk 2 above)
Comment (jvandaal) [reply src/Be.Vlaanderen.Basisregisters.AcmIdm/AddAuthorizationExtensions_WebApi.cs]: Changed to a dash.

File: src/Be.Vlaanderen.Basisregisters.AcmIdm/AddAuthorizationExtensions_MinimalApi.cs (hunk 1 above)
Comment (jvandaal) [reply src/Be.Vlaanderen.Basisregisters.AcmIdm/AddAuthorizationExtensions_MinimalApi.cs]: Changed to a dash.

**Pull request merged**